app-modules/knowledge-base/tests/KnowledgeBaseCategory/EditKnowledgeBaseCategoryTest.php

```diff
@@ -140,3 +145,28 @@
 
     assertEquals($request['name'], $knowledgeBaseCategory->fresh()->name);
 });
+
```

Comment:
Please add a test ensuring that only expected records show up as selectable options for the attach